pkg/sessions/redis/redis_store_test.go

```diff
@@ -20,10 +21,18 @@ import (
 
 const redisPassword = "0123456789abcdefghijklmnopqrstuv"
 
+type wrappedRedisLogger struct {
```

Comment:
Might be worth a small note here what the purpose of this is
```suggestion
// wrappedRedisLogger wraps a logger so that we can coerce the logger to
// fit the expected signature for go-redis logging
type wrappedRedisLogger struct {
```